test/support/api_conn_case.ex

```diff
@@ -0,0 +1,28 @@
+defmodule GoogleCrawlerWeb.ApiConnCase do
+  use ExUnit.CaseTemplate
+
+  using do
+    quote do
+      alias GoogleCrawlerWeb.ApiRouter.Helpers, as: Routes
+
+      use Mimic
```

Comment:
I guess we don't need Mimic yet

Reply:
To align with the `conn_case` and we will use Mimic in `keyword_controller_test`